shell/browser/native_window_views.cc

```diff
@@ -757,6 +759,38 @@ void NativeWindowViews::SetAlwaysOnTop(bool top,
     NativeWindow::NotifyWindowAlwaysOnTopChanged();
 
   widget()->SetAlwaysOnTop(top);
+
+#if defined(OS_WIN)
+  // Re-use level names from MacOS to have the same electron api.
+  if (top) {
+    bool above_task_bar = true;
```

Comment:
We can probably make this logic cleaner by considering which `level` values change the default; something like: 
```cpp
if (top) {
    bool above_task_bar = true;
    std::vector<std::string> levels = {"floating", "torn-off-menu", "modal-panel", "main-menu", "status"};
    if (std::find(levels.begin(), levels.end(), value) != levels.end()) {
      above_task_bar = false;
    }
    
    if (!above_task_bar) {
      // Behind the taskbar.
      const HWND task_bar_hwnd = ::FindWindow(kUniqueTaskBarClassName, NULL);
      ::SetWindowPos(GetAcceleratedWidget(), task_bar_hwnd, 0, 0, 0, 0,
                    SWP_NOMOVE | SWP_NOSIZE | SWP_NOACTIVATE);
    }
}
```

We don't need to specifically check for the final three values ("pop-up-menu", "screen-saver", "dock") since they wouldn't change the default value you've already set at the top of the conditional.

Reply:
Done

Reply:
Thx a lot for the suggestion.